package.json

```diff
@@ -86,20 +98,29 @@
     "rimraf": "2.5.4",
     "sass": "^1.37.4",
     "semantic-release": "^17.0.4",
-    "sinon": "^1.17.4"
+    "sinon": "^1.17.4",
+    "tsd": "^0.17.0",
+    "typescript": "^4.3.5"
   },
   "scripts": {
     "prebuild": "yarn run clean",
-    "build": "BABEL_ENV=production babel src -d dist && cp -r ./src/*.png ./dist && sass ./src/intlTelInput.scss ./dist/main.css",
+    "build": "yarn compile:js && yarn compile:dts && yarn compile:css && yarn compile:png",
+    "compile:js": "BABEL_ENV=production babel src -d dist",
+    "compile:dts": "rsync -avh --include='*/' --include='*.d.ts' --exclude='*' src/ dist --prune-empty-dirs",
+    "compile:css": "sass ./src/intlTelInput.scss ./dist/main.css",
+    "compile:png": "cp -r -v ./src/*.png ./dist",
     "clean": "rimraf dist",
     "website:start": "yarn workspace website run start",
     "website:dryrun": "yarn workspace website run deploy:dryrun",
     "website:deploy": "yarn workspace website run deploy",
-    "lint": "eslint src website/.storybook *.js",
+    "lint": "eslint src website/.storybook --ext .js,.d.ts",
```

Comment:
Lint everything in one pass. No need to lint `.js` and `.d.ts` files separately.